packages/flutter_tools/test/general.shard/web/chrome_test.dart

```diff
@@ -163,3 +167,21 @@ void main() {
 }
 
 class MockOperatingSystemUtils extends Mock implements OperatingSystemUtils {}
+
+Future<Chrome> testLaunchChrome(String userDataDir, FakeProcessManager processManager, ChromeLauncher chromeLauncher) {
```

Comment:
Thanks!

This is much simpler to read since it does not depend on the specific location in the test script